### ISSUE TITLE
[pandas] Sorting a table from a pandas-loader supported format raises an error

### ISSUE THREAD

**Issue by jtrakk** [saulpw/visidata#478]
**Small description**
Sorting a table from a parquet file raises an error.
**Expected result**
The table is sorted.
**Actual result with screenshot**
If you get an unexpected error, please include the full stack trace that you get with `Ctrl-E`.

```py
 Traceback (most recent call last):                                                                                                            ‖
   File "/home/user/tmp/venv/lib/python3.6/site-packages/visidata/vdtui.py", line 650, in toplevelTryFunc                                      ‖
     ret = func(*args, **kwargs)                                                                                                               ‖
   File "/home/user/tmp/venv/lib/python3.6/site-packages/visidata/vdtui.py", line 1568, in orderBy                                             ‖
     self.rows.sort(key=lambda r,cols=cols,prog=prog: prog.addProgress(1) and tuple(c.getTypedValueNoExceptions(r) for c in cols), **kwargs)   ‖
   File "/home/user/tmp/venv/lib/python3.6/site-packages/visidata/loaders/_pandas.py", line 17, in __getattr__                                 ‖
     return getattr(self.df, k)                                                                                                                ‖
   File "/home/user/tmp/venv/lib/python3.6/site-packages/pandas/core/generic.py", line 5274, in __getattr__                                    ‖
     return object.__getattribute__(self, name)                                                                                                ‖
 AttributeError: 'DataFrame' object has no attribute 'sort'        
```

**Steps to reproduce with sample data and a .vd**

Create a parquet file.

```py
import pandas as pd
pd.DataFrame({'x': [1,2,3]}).to_parquet('tmp.parquet')
```

Read the parquet file into visidata.

```
vd -f pandas tmp.parquet
```

Sort the table by pressing `[`.

**Additional context**
Please include the version of VisiData.

VisiData v1.5.2

**Comment by anjakefala** [comment]
Thanks @jtrakk!

**Comment by anjakefala** [comment]
Fixed in #496!
